plugins/subm.yaml

```diff
@@ -1,9 +1,12 @@
+# This template is generated by krew-release-bot-helper
```

Comment:
I'm assuming you added this manually?

Reply:
No this was generated by the [krew-release-bot](https://github.com/rajatjindal/krew-release-bot) GHA.